### PR TITLE
Update dependancies

### PR DESCRIPTION
**Pull request by richmccartney**
#### What's this PR do?
Updates all dependancies to the latest version, removes compass and replaces it with gulp

#### Where should the reviewer start?
Pulling down the branch ```update-dependancies``` then running ```npm install```

#### How should this be manually tested?
Running a gulp task should return SCSS lint errors 

#### Any background context you want to provide?
_n/a_

#### What are the relevant tickets?
_n/a_

#### Screenshots (if appropriate)
_n/a_

#### Questions:
_n/a_

### CODE REVIEW

**Comment by anthonydillon** [comment]
Can you remove the npm-debug.log please. It's added to the ignore but needs to be removed before the feature branch.

File: package.json [hunk 1]
```diff
@@ -1,15 +1,74 @@
 {
-  "name": "canonical-landscape",
-  "version": "0.0.1",
-  "description": "Static Landscape ",
+  "author": {
+    "name": "Canonical Webteam",
```
Comment (anthonydillon) [inline package.json]: Think `Landscape website` is a better name

File: package.json [hunk 2]
```diff
@@ -1,15 +1,74 @@
...
+    "email": "webteam@canonical.com",
+    "url": "http://design.canonical.com/team"
```
Comment (anthonydillon) [inline package.json]: `landscape.canonical.com`

File: package.json [hunk 3]
```diff
@@ -1,15 +1,74 @@
...
+  },
+  "contributors": [
```
Comment (anthonydillon) [inline package.json]: Just you @richmccartney at the moment so update to reflect this please

File: package.json [hunk 4]
```diff
@@ -1,15 +1,74 @@
...
+      "name": "Richard McCartney",
+      "email": "richard.mccartney@canonical.com"
+    },
+    {
+      "name": "Peter Mahnke",
+      "email": "peter.mahnke@canonical.com"
+    }
+  ],
+  "description": "Landscape static templates",
+  "homepage": "landscape.canonica.com",
```
Comment (anthonydillon) [inline package.json]: landscape.canonica**l**.com

**Comment by anthonydillon** [comment]
Few comments in code and hitting a QA issue:
$ npm install
npm WARN package.json landscape.canonical.com@ homepage field must start with a protocol.

> landscape.canonical.com@ prepublish /home/ant/Documents/Canonical/Git/landscape.canonical.com
> ./.install_dependencies.sh

sh: 1: ./.install_dependencies.sh: not found

**Comment by richmccartney** [comment]
@anthonydillon if you take a look now, the dependancies should be fixed and the changes required here.

**Comment by anthonydillon** [comment]
Can you remove the `npm-debug.log` file please. Its adding 20k line to the repo.

File: package.json [hunk 5]
```diff
@@ -1,16 +1,72 @@
...
+    {
+      "name": "Peter Mahnke",
+      "email": "peter.mahnke@canonical.com"
+    },
+    {
+      "name": "Carla Berkers",
+      "email": "Carla.berkers@canonical.com"
+    },
+    {
+      "name": "Peter Mahnke",
```
Comment (anthonydillon) [inline package.json]: Peter is here twice.

**Comment by richmccartney** [comment]
@anthonydillon I thought I removed it must have snuck back in. All done now.

**Comment by anthonydillon** [comment]
LGTM :+1:

**Pull request merged**